rest_api/understanding-api-support-tiers.adoc

```diff
@@ -0,0 +1,16 @@
+[id="understanding-api-support-tiers"]
+= Understanding API support tiers
+include::modules/common-attributes.adoc[]
+:context: understanding-api-support-tiers
+
+toc::[]
+
+Support tiers for bare-metal configurations also apply to virtualized configurations except for any feature that directly interacts with hardware.  Those features directly related to hardware have no application operating environment (AOE) compatibility level beyond that which is provided by the hardware vendor.  For example, applications that rely on Graphics Processing Units (GPU) features are subject to the AOE compatibility provided by the GPU vendor driver.
+
+Support tiers in a cloud environment for cloud specific integration points have no API or AOE compatibility level beyond that which is provided by the hosting cloud vendor.  For example, APIs that exercise dynamic management of compute, ingress, or storage are dependent upon the underlying API capabilities exposed by the cloud platform.  Where a cloud vendor modifies a prerequisite API, Red Hat will provide commercially reasonable efforts to maintain support for the API with the capability presently offered by the cloud infrastructure vendor.
```

Comment:
We should try an link / connect our API's to the Cloud Provider API documentation. 

Ideally we should be able to map to the provider api's and show a chain of 'trust'/'change management' with these statement.